src/backend/distributed/planner/query_pushdown_planning.c

```diff
@@ -194,6 +200,29 @@ JoinTreeContainsSubquery(Query *query)
 }
 
 
+bool
```

Comment:
Missing function comment. Also, There is no regression test that triggers `if (rte->rtekind == RTE_RESULT)`, Can you find/add one?

Reply:
Ping on this

Reply:
No tests in `check-multi` run into this. I'm wondering if postgres has optimized out `RTE_RESULT` by time it gets to us